.nux/nux.yaml

```diff
@@ -0,0 +1 @@
+seen: 1
```

Comment:
is this from dagster? https://docs.dagster.io/deployment/oss/dagster-yaml#nux

if so, we should put this path onto our gitignores and dockerignores in https://github.com/robert-koch-institut/mex-template